src/managers/GuildMemberManager.js

```diff
@@ -246,17 +246,16 @@ class GuildMemberManager extends CachedManager {
     }
 
     const endpoint = this.client.api.guilds(this.guild.id).prune;
-
+    let pruned;
     if (dry) {
-      return endpoint.get({ query, reason }).then(data => data.pruned);
-    }
-
-    return endpoint
-      .post({
+      ({ pruned } = await endpoint.get({ query, reason }));
+    } else {
+      ({ pruned } = await endpoint.post({
         data: { ...query, compute_prune_count },
         reason,
-      })
-      .then(data => data.pruned);
+      }));
+    }
```

Comment:
This could probably be written as:
```javascript
const { pruned } = await (dry
  ? endpoint.get({ query, reason })
  : endpoint.post({
      data: { ...query, compute_prune_count },
      reason
    }));
```